### ISSUE TITLE
Missing version in help commandlet output

### ISSUE THREAD

**Issue by mvomiero** [devonfw/IDEasy#329]
### Expected behavior

As a IDEasy user, I want to know the current IDEasy version when using the `help` commandlet

### Actual behavior

the version is missing:
<img width="322" alt="ide help" src="https://github.com/devonfw/IDEasy/assets/102261609/c644abbb-87a7-44e7-af32-b513eb561eef">

correct output should be for example: 
```
Current version of IDE is 2024.05.001-alpha-SNAPSHOT
```

### Steps to reproduce

type `ide help`

**Comment by hohwille** [comment]
@mvomiero thanks for creating this bug issue.
For the record to be more precise it is that the version is not properly computed in or included into the GraalVM native image (manifest).
This is the code that determines the version:
https://github.com/devonfw/IDEasy/blob/b2754e933b89db43d9a661f451691b469c12672b/cli/src/main/java/com/devonfw/tools/ide/version/IdeVersion.java#L32-L43

The version is printed in the help but also printed when `ide -v` or `ide --version` is invoked.
So the help itself is printing the version and nothing needs to be changed there but the bug is that even in a release of IDEasy the `IdeVersion` class determines `SNAPSHOT` as version what is actually incorrect.

Maybe we just need to configure some additional flag to our graalvm build and the problem will be fixed.
Or we have some bug that the version is not correctly added to our `MANIFEST.MF` file via maven `pom.xml`.
https://github.com/devonfw/IDEasy/blob/b2754e933b89db43d9a661f451691b469c12672b/cli/pom.xml#L103-L107

**Comment by hohwille** [comment]
I just build a `JAR` of our IDEasy CLI and unpacked it. The `MANIFEST.MF` looks like this:
```
Manifest-Version: 1.0
Created-By: Maven JAR Plugin 3.3.0
Build-Jdk-Spec: 17
Class-Path: lib/commons-compress-1.23.0.jar lib/javax.json-api-1.1.4.jar
  lib/javax.json-1.1.4.jar lib/jackson-databind-2.14.1.jar lib/jackson-a
 nnotations-2.14.1.jar lib/jackson-core-2.14.1.jar lib/jackson-datatype-
 jsr310-2.14.1.jar lib/jackson-dataformat-xml-2.14.1.jar lib/stax2-api-4
 .2.1.jar lib/woodstox-core-6.4.0.jar lib/slf4j-api-2.0.12.jar lib/logba
 ck-classic-1.5.3.jar lib/logback-core-1.5.3.jar lib/progressbar-0.10.0.
 jar lib/jline-3.24.1.jar lib/jansi-2.4.0.jar
Specification-Title: ide-cli
Specification-Version: 0.0
Specification-Vendor: devonfw
Implementation-Title: ide-cli
Implementation-Version: 2024.04.001-alpha-SNAPSHOT
Implementation-Vendor: devonfw
Main-Class: com.devonfw.tools.ide.cli.Ideasy
```

So as we can see the implementation version is `2024.04.001-alpha-SNAPSHOT` what seems correct (my fork is not synced) but at least we should not get `SNAPSHOT` as version and it seems more like a `GraalVM` issue reading from the `MANIFEST.MF` in the native image.

**Comment by hohwille** [comment]
Just to confirm that the Java implementation is working:
```
$ java -cp "target/ide-cli-2024.04.001-alpha-SNAPSHOT.jar;lib/jline-3.24.1.jar;lib/commons-compress-1.23.0.jar;lib/javax.json-api-1.1.4.jar" com.devonfw.tools.ide.cli.Ideasy help
__       ___ ___  ___
\ \     |_ _|   \| __|__ _ ____ _
 > >     | || |) | _|/ _` (_-< || |
/_/ ___ |___|___/|___\__,_/__/\_, |
   |___|                       |__/

Current version of IDE is 2024.04.001-alpha-SNAPSHOT
...
```

So it is definitely a GraalVM native image problem.

**Comment by jan-vcapgemini** [comment]
We could try this: https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/Resources.md
As we are currently including the nls folder in this way too, we could do the same for the MAINFEST.
We just need to move the MANIFEST to our resources too.

**Comment by hohwille** [comment]
I already fixed this with PR #368. Nice.
